docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -78,45 +78,85 @@ These instructions assume you have experience using the command line, understand
 >
 > If debugging stops working; for example, if breakpoints are being ignored; stop debugging. Then, if necessary, close all host application windows and the Node window. Finally, close Visual Studio Code and reopen it.
```

Comment:
```suggestion
> If debugging stops working---for example, if breakpoints are being ignored---stop debugging. Then, if necessary, close all host application windows and the Node window. Finally, close Visual Studio Code and reopen it.
```
Use em dash instead of semicolon here! The triple hyphen is supposed to render as em dash in GitHub flavored markdown. Alternatively, you could insert HTML here. Both **&mdash;**  and  **&#151;** render an em dash.